archfx_cloud/scripts/query_factory_map.py

```diff
@@ -0,0 +1,252 @@
+"""
+    ADT-287  Example code for creating a full factory map for flex.
+"""
+import os
+from cryptography.fernet import Fernet
+from typing import Dict, List
+import pickle
+import pandas as pd
+from archfx_cloud.api.connection import Api
+
+DEBUG = False
+
+
+class QueryFactoryMap:
+    def __init__(self, email: str):
+        self._email = email
+        self.token_filename = ".cloud_api_token"
+        self.__initialize_connection()
+        self._sites = {}
+        self._tree = {}
+        self._area = {}
+        self._line = {}
+        self._devices = {}
+
+    def __initialize_connection(self):
+        # api = Api('https://arch.archfx.io')
+        api = Api("https://flex.archfx.io")
+
+        passwd = self.get_password()
+        ok = api.login(email=self._email, password=passwd)
+        if ok:
+            print("logged in successfully!")
+            self.api = api
+            # api.logout()
```

Comment:
You rewrote this but it's already provided by `BaseMain` from `archfx_cloud` package.